scaladoc-js/src/Main.scala

```diff
@@ -1,22 +0,0 @@
-package dotty.tools.scaladoc
-
-object Main:
```

Comment:
What will be the impact on other projects that use scaladoc? Do we have end-to-end tests that check that they will still work after the PR is merged?

Reply:
There are none such test sadly, however, we have two projects that can be checked if everything is working ok:
https://scala3doc.virtuslab.com/pr-new-frame-poc/testcases/index.html
https://scala3doc.virtuslab.com/pr-new-frame-poc/self/index.html

Reply:
Great, is this now using the PR?

Reply:
Yes, you can see the link. Actually, each PR that is named with `scaladoc/` prefix (e. g.) `scaladoc/branch-name` will be available under https://scala3doc.virtuslab.com/pr-branch-name/index.html. The branch must be on `dotty-staging` or on your fork with azure credentials set up

Reply:
Nice! But AFAICT, those two examples don’t use the static website?

Reply:
Oh, yes, they don't. I can add some basic static site for testing purposes